### ISSUE TITLE
[Bug]: Accessors / mutators not generated in Eloquent helpers code if method is `protected`

### ISSUE THREAD

**Issue by aguingand** [laravel-idea/plugin#565]
### Bug description

In the [docs](https://laravel.com/docs/9.x/eloquent-mutators#defining-an-accessor), accessors / mutators methods are `protected` but _Generate Eloquent helpers code_ only generate phpdoc for the attribute if the method is `public`.

### Plugin version

5.4.0.221

### Operating system

MacOS

### Steps to reproduce

- Create following model with accessor
```php
<?php
 
namespace App\Models;
 
use Illuminate\Database\Eloquent\Casts\Attribute;
use Illuminate\Database\Eloquent\Model;
 
class User extends Model
{
    /**
     * Get the user's first name.
     *
     * @return \Illuminate\Database\Eloquent\Casts\Attribute
     */
    public function firstName(): Attribute
    {
        return Attribute::make(
            get: fn ($value) => ucfirst($value),
        );
    }
}
```
- Run _Generate Eloquent helpers code_, the attribute `first_name` is generated
- Pass the method `protected` like in the [doc](https://laravel.com/docs/9.x/eloquent-mutators#defining-an-accessor)
- Run _Generate Eloquent helpers code_, the attribute `first_name` is not generated

**Comment by adelf** [comment]
It is fixed in the 5.5. Please update.

**Comment by aguingand** [comment]
Ok thanks, it's working now !
